### PR TITLE
Enable color input for regions (#14)

### PR DESCRIPTION
**Pull request by johnyrufus**

### CODE REVIEW

File: cartogram_generator/fill_with_density.c [hunk 1]
```diff
@@ -298,7 +300,10 @@ BOOLEAN fill_with_density1 (char *map_file_name, char *area_file_name,
   while (fgets(line, MAX_STRING_LENGTH, area_file) != NULL) {
     id = -1;
     area = -1.0;
-    sscanf(line, "%d%*[, ]%lf", &id, &area);
+    r = 0.96;  
+    g = 0.92;
+    b = 0.70;
```
Comment (mgastner) [inline cartogram_generator/fill_with_density.c]: After dividing by 255 in lines 315-317, the fill colors are essentially black, so the default fill color (black) is indistinguishable from the default border color (also black). Please change to r = 245.0, g = 235.0, b = 179.0.

**Review by mgastner — CHANGES_REQUESTED** [review]
Please make default values for r, g, b larger.

**Comment by johnyrufus** [comment]
Thanks for pointing this out. I modified the default values and tested it as well to see if the default colors come out as before, when the input color fields are not present.

**Comment by mgastner** [comment]
Thank you for adding the color option!

**Pull request merged**